From: Jonas Meurer <jonas@freesources.org>
Date: Thu, 21 Sep 2017 14:52:36 +0200
Subject: Remove embedded remote images from README.rst

 The documentation files shipped with Debian packages should not embed
 remote images, especially not from sources that are known to track
 users (like shields.io).

Forwarded: not-needed
Author: Jonas Meurer <jonas@freesources.org>
---
 README.rst | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/README.rst b/README.rst
index a706c11..a25488a 100644
--- a/README.rst
+++ b/README.rst
@@ -2,18 +2,6 @@
 Mailman - The GNU Mailing List Management System
 ================================================
 
-.. image:: https://gitlab.com/mailman/mailman/badges/master/build.svg
-    :target: https://gitlab.com/mailman/mailman/commits/master
-
-.. image:: https://readthedocs.org/projects/mailman/badge
-    :target: https://mailman.readthedocs.io
-
-.. image:: https://img.shields.io/pypi/v/mailman.svg
-    :target: https://pypi.org/project/mailman/
-
-.. image:: https://img.shields.io/pypi/dm/mailman.svg
-    :target: https://pypi.org/project/mailman/
-
 Copyright (C) 1998-2019 by the Free Software Foundation, Inc.
 
 This is GNU Mailman, a mailing list management system distributed under the
